src/main.rs

```diff
@@ -17,18 +17,45 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+use clap::{CommandFactory, ErrorKind, Parser};
+
 use astarte_device_sdk::options::AstarteOptions;
 use astarte_device_sdk::registration;
 use astarte_device_sdk::AstarteDeviceSdk;
 
 use astarte_message_hub::config::MessageHubOptions;
 use astarte_message_hub::error::AstarteMessageHubError;
 
+/// A central service that runs on (Linux) devices for collecting and delivering messages from N
+/// apps using 1 MQTT connection to Astarte.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
```

Comment:
Could we rename this into something different, since clap has a derive macro and trait/struct named `Args`?

I propose something like `Cli` which is also used in more recent versions of the clap documentation.